topics/ajax/index.md

```diff
@@ -0,0 +1,9 @@
+---
+display_name: AJAX
+short_description: Ajax is a technique for creating better, faster, and more interactive web applications.
+topic: ajax
+related: json, html, xml, javascript
+wikipedia_url: https://en.wikipedia.org/wiki/Ajax_(programming)
+---
+AJAX stands for asynchronous JavaScript and XML. It is collection of several web technologies including HTML, CSS, JSON, XML, JavaScript. It is used for creating dynamic web pages in which small part of web page changes without reloading the page. It is a web browser technology independent of web server software.
```

Comment:
- Remember to add ", and" between XML and JavaScript!
- "part of the web page changes" > "parts of the web page change"
- Remove the last sentence.